standalone/runtime/src/lib.rs

```diff
@@ -1016,7 +1016,13 @@ impl pallet_treasury::Config for Runtime {
     type SpendFunds = Bounties;
     type WeightInfo = pallet_treasury::weights::SubstrateWeight<Runtime>;
     type MaxApprovals = MaxApprovals;
-    type SpendOrigin = frame_system::EnsureWithSuccess<EnsureRoot<AccountId>, AccountId, MaxBalance>;
+    type SpendOrigin = frame_system::EnsureWithSuccess<
```

Comment:
Khala and Phala need to change like this also?

Reply:
https://github.com/Phala-Network/khala-parachain/issues/265